samza-core/src/main/java/org/apache/samza/storage/StorageManagerUtil.java

```diff
@@ -45,7 +45,9 @@
 
 public class StorageManagerUtil {
```

Comment:
Are there tests for this class?

Reply:
No its tested from its consumers -- task-storage-manager, and sideinputstorage manager